src/Components/Components/test/CascadingParameterTest.cs

```diff
@@ -312,16 +312,16 @@ public void CascadingValueThrowsIfFixedFlagChangesToTrue()
                 builder.CloseComponent();
             });
             renderer.AssignRootComponentId(component);
-            component.TriggerRender();
+            var _ = renderer.Invoke(() => component.TriggerRender());
 
             // Act/Assert
             isFixed = true;
-            var ex = Assert.Throws<InvalidOperationException>(() => component.TriggerRender());
+            var ex = await Assert.ThrowsAsync<InvalidOperationException>(() => renderer.Invoke(() => component.TriggerRender()));
```

Comment:
This is just because xUnit complains about Throw with Func<Task>. I will replace this for void RunHelper(Func<Task> t) so that we can maintain `Assert.Throw(() => renderer.Invoke(() => component.TriggerRender())`

Reply:
This is fixed now by doing renderer.Invoke inside TriggerRender